tests/ConnectionTest.php

```diff
@@ -396,6 +397,48 @@ public function testConnectStartsTransactionInNoAutoCommitMode(): void
         self::assertTrue($conn->isTransactionActive());
     }
 
+    public function testTransactionIsNotActiveAfterTransactionalInAutoCommitMode(): void
+    {
+        $driverMock = $this->createMock(Driver::class);
```

Comment:
That's a stub since you are not asserting anything on it.

```suggestion
        $driverMock = $this->createStub(Driver::class);
```

Reply:
I'm not familiar with phpunit, I copied it from other tests. Changed to stub, thanks.